lib/elixir/lib/supervisor.ex

```diff
@@ -377,7 +377,7 @@ defmodule Supervisor do
     the supervisor will automatically shut down its children, then itself.
 
   Only `:transient` and `:temporary` child processes can be marked as significant,
-  and this configuration affects the behavior. Significant `:transient` child
+  and this configuration affects the behaviour. Significant `:transient` child
```

Comment:
In this context, I don't think it is referring to the OTP [behaviour](https://hexdocs.pm/elixir/typespecs.html#behaviours) (British spelling by convention) but just in the regular sense of the word
```suggestion
  and this configuration affects the behavior. Significant `:transient` child
```